src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/ReadyToRunTableManager.cs

```diff
@@ -50,8 +50,6 @@ protected virtual void Graph_NewMarkedNode(DependencyNodeCore<NodeFactory> obj)
         {
             IMethodBodyNode methodBodyNode = obj as IMethodBodyNode;
             var methodNode = methodBodyNode as IMethodNode;
-            if (methodNode == null)
```

Comment:
Why is this getting deleted?

Reply:
`obj as ShadowConcreteMethodNode` below always evaluates to `null` in this codebase. There's no `ShadowConcreteMethodNode` because we [don't model generics that way](https://github.com/dotnet/coreclr/pull/26927#discussion_r329488486). This was copy-pasted from the full AOT compiler.

If/when we start modeling generics we can add this back (or not, depending on how we do it) - but it's unreachable code right now.

Reply:
Ah, makes sense. Thanks for the clarification